linode/manager
# change: [M3-8373] - Fix unexpected heading spacing on Account Maintenance page

## Description 📝
On the Account Maintenance page, the "Pending" and "Completed" headings have spacing which is inconsistent with the spacing of our headings on other pages.

## Changes  🔄
- Account maintenance headings are aligned to the tables that they're labeling

## Target release date 🗓️
10/28

## Preview 📷

| Before  | After   |
| ------- | ------- |
|![image](https://github.com/user-attachments/assets/71ab440c-894f-47b5-9c28-20f6f04bbbe1) | ![image](https://github.com/user-attachments/assets/68fd94de-f248-4299-b6bf-2327ec9f2be6)|

## How to test 🧪

### Verification steps
- Enable MSW
- Navigate to `localhost:3000/account/maintenance`
- Verify that the table headings are aligned with the table rows and have uniform padding.

## As an Author I have considered 🤔

*Check all that apply*
